src/api/tests/siteConvertingHelpers.spec.ts

```diff
@@ -60,20 +60,16 @@ describe('Sharing Permission Helper Tests', () => {
         // eslint-disable-next-line camelcase
         client_count: 1,
       } as SiteDTO;
-      const convertedType = convertSiteToAvailableParticipantDTO(site, participantTypes, []);
+      const convertedType = convertSiteToSharingSiteDTO(site, []);
```

Comment:
nit: just a reminder, I think test names need to be updated later,
we could remove tests like it('should map client type to participant type'

Reply:
Yeah, I'll make a note to come back later and revisit some of the test descriptions.